pkg/metrics/metrics.go

```diff
@@ -45,6 +55,13 @@ var (
 		},
 		[]string{opSrcLabel},
 	)
+	customResourceGaugeVec = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "custom_resource_total",
+			Help: "A counter that stores information about the presense of a custom OperatorSource or any CatalogSourceConfig in the cluster.",
```

Comment:
This is a Gauge, not a Counter, which are different types in Prometheus.

Reply:
Right, I was using a counter before I decided to change to a gauge instead but forgot to change the comment. Thank you for noticing it